services/sandbox/controller_service.go

```diff
@@ -0,0 +1,93 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package sandbox
+
+import (
+	"context"
+	"errors"
+
+	api "github.com/containerd/containerd/api/services/sandbox/v1"
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/plugin"
+	"github.com/containerd/containerd/services"
+	"google.golang.org/grpc"
+)
+
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type: plugin.GRPCPlugin,
+		ID:   "controllers",
```

Comment:
`ID: "sandbox-controllers"`? Controller is a generic term. I'm unsure we can take that for sandbox.

"Controller" is also confusing for Kubernetes folks due to https://kubernetes.io/docs/concepts/architecture/controller/, but I really don't have a better name...

Reply:
Renamed to "sandbox-controllers". Here it meant to be "control runtime sandbox instances". I agree that it's popular word and used in many other places, so I'm open to suggestions here.